### PR TITLE
Captions experiment 580

### PR DESCRIPTION
**Pull request by mccalluc**
@afred: thoughts? Towards #580.

### CODE REVIEW

**Comment by mccalluc** [comment]
I could imagine doing some interpolation to break up the larger chunks, but this is something, and if it gets in the way, user doesn't need to turn it on.

**Comment by mccalluc** [comment]
@Muraszko: Currently splitting on every sentence. In some instances, they do go by too fast. I would suggest deploying it and, if it's worth while, getting more eyes looking at it to determine if the behavior needs to be tweaked.

File: spec/models/captioner_spec.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require_relative '../../lib/captioner'
+
+describe Captioner do
+  it 'produces WebVTT' do
+    tei = File.read(__dir__ + '/../fixtures/transcript/mock-transcript.xml')
+    expect(Captioner.from_tei(tei)).to eq(File.read(__dir__ + '/../fixtures/transcript/mock-transcript.vtt'))
+  end
+end
```
Comment (afred) [inline spec/models/captioner_spec.rb]: Can you wrap unit tests in `describe method_name`? It makes it more explicit about exactly what is being tested. E.g...

```ruby
describe Captioner do
  describe '.from_tei' do
    it 'produces WebVTT' do
```

File: spec/models/transcripter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require_relative '../../lib/transcripter'
+
+describe Transcripter do
+  it 'produces HTML' do
+    tei = File.read(__dir__ + '/../fixtures/transcript/mock-transcript.xml')
+    expect(Transcripter.from_tei(tei)).to eq(File.read(__dir__ + '/../fixtures/transcript/mock-transcript.html'))
+  end
+end
```
Comment (afred) [inline spec/models/transcripter_spec.rb]: Can you wrap unit tests in `describe method_name`? It makes it more explicit about exactly what is being tested. E.g...

```ruby
describe Transcipter do
  describe '.from_tei' do
    it 'produces HTML' do
```

**Comment by afred** [comment]
@mccalluc thanks for tweaking those specs. This is a good idea! Interested to see how it turns out.

**Pull request merged**